s/elements/bindings-editor/utils/loadBindings.ts

```diff
@@ -1,11 +1,28 @@
+import {StateSetter} from "@chasemoskal/magical/x/view/types.js"
 import {Bindings} from "../../../types.js"
 import { defaultBindings } from "../../context/parts/default-bindings.js";
 
-export function loadBindings() {
+export function listenForLocalStorageBindings(setBindings: StateSetter<Bindings>) {
+	window.addEventListener('storage', () => {
+		const savedBindings = localStorage?.getItem('bindings')!;
+		const savedBindingsJSON:Bindings = JSON.parse(savedBindings)
+		setBindings(savedBindingsJSON)
+	})
+}
+
+export function loadLocalStorageBindings() {
 	const savedBindings = localStorage?.getItem('bindings')!;
-	const savedBindingsJSON: Bindings = JSON.parse(savedBindings)
+	const savedBindingsJSON:Bindings = JSON.parse(savedBindings)
```

Comment:
![image](https://user-images.githubusercontent.com/62626145/203662386-a296a58b-d6d3-4e6f-bcac-efe44a2fffdf.png)

just curious as to why the space between the variable name and the type was removed
it's kinda more readable when it's spaced, yeah?

Reply:
a space would be preferable here -- but it's no big deal

we're in an early stage of development -- one day in the future, we'll apply more rigid style rules and linters and such

in the past i've been too demanding in terms of syntactical perfection :D